docs/oef-ledger.md

```diff
@@ -12,7 +12,33 @@ The 'Open Economic Framework' (OEF) consists of protocols, languages and market
   <p>The OEF is under development. Expect frequent changes. What follows is a description of the current implementation.</p>
 </div>
 
-At present, the OEF services are fulfilled by an `OEF search and communication node`. This node consists of two parts. A `search node` part enables agents to register their services and search and discover other agents' services. A `communication node` part enables agents to communicate with each other.
+At present, the OEF's capabilities are fulfilled by two components:
+
+- a permissionless, public peer to peer (agent to agent) communication network, called the Agent Communication Network; and
+- a centralized search and discovery system.
+
+The latter will be decentralized over time.
```

Comment:
ok, I've seen it only now, I think you can ignore the comment above re. the ambiguity of the first OEF definition.